src/Microsoft.ApplicationInsights/TelemetryClient.cs

```diff
@@ -60,10 +59,12 @@ public TelemetryClient(TelemetryConfiguration configuration)
         /// </summary>
         public TelemetryContext Context
         {
-            get { return LazyInitializer.EnsureInitialized(ref this.context, () => new TelemetryContext()); }
-            internal set { this.context = value; }
+            get;
```

Comment:
who will initialize it?

Reply:
due to formatting rules, see few lines below.